### PR TITLE
notmyidea-cms: Corrected header hierarchy and URLs.

### PR DESCRIPTION
**Pull request by Eric-Sparks**

### CODE REVIEW

**Comment by Eric-Sparks** [comment]
Fixed two problems (and tested them) that were affecting markup testing and https-only testing.

**Comment by Eric-Sparks** [comment]
Can this be merged?

File: notmyidea-cms/templates/index.html [hunk 1]
```diff
@@ -6,20 +6,20 @@
...
-                <h1 class="entry-title"><a href="{{ SITEURL }}/{{ article.url
-                }}">{{ article.title }}</a></h1>
+                <h2 class="entry-title"><a href="{{ SITEURL }}/{{ article.url
+                }}">{{ article.title }}</a></h2>
                 {% include 'article_infos.html' %}<!-- /.post-info -->
                 {{ article.content }}
         </article></aside><!-- /#featured -->
         {% if loop.length  > 1 %}
-                <h1>Other articles</h1>
+	<!--                <h2>Other articles</h2> -->
```
Comment (justinmayer) [inline notmyidea-cms/templates/index.html:15]: I don't think committing commented-out code to the repository is optimal practice. Wouldn't you agree?

**Review by justinmayer — CHANGES_REQUESTED** [review]
Nice work! Other than one line that contains commented-out code, I think this looks good.

File: notmyidea-cms/templates/index.html (hunk 1 above)
Comment (Eric-Sparks) [reply notmyidea-cms/templates/index.html:15]: Umm, no it shouldn't.  Why did I do this???  Ugh, it's been too long.

**Comment by Eric-Sparks** [comment]
Github sometimes does not agree with me.  This is one of those times.  I created a new PR for this change *including* the requested fix.
Duplicate of #729

**Pull request closed without merging**